app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -38,6 +41,7 @@
 public class MainActivity extends AppCompatActivity
 {
     private static final String TAG = "LoyaltyCardLocker";
+    private Menu _menu;
```

Comment:
The style in the project is to not have a leading _ in front of members. You can change this to 'menu'.